pre-backup

```diff
@@ -403,19 +373,21 @@ check_version() {
 }
 
 # Collect all output
-exec 9> >(exec ts '%FT%T%z' > "${output_dir}/log")
-exec > >(exec tee -a /dev/fd/9 >&1)
-exec 2> >(exec tee -a /dev/fd/9 >&2)
+## save original stdout for dumping tar in the end
```

Comment:
@simu What was the idea here?

Reply:
To capture the log of the backup script in a file which is part of the backup itself, IIRC

Reply:
So for this use case, we do not need that anymore. Objections if I drop those lines completely?

Reply:
yeah should be fine to drop commented-out code. We can always reintroduce it if necessary.